.travis.yml

```diff
@@ -41,10 +41,20 @@ jobs:
       env: aarch64
       arch: arm64
       script:
-        - sed -i 's/java_arch=32-bit$/java_arch=64-bit/' build-image/openhabian.conf
+        - sed -i 's/java_opt=Zulu32$/java_opt=Zulu64/' build-image/openhabian.conf
```

Comment:
use same ^java_opt=.* in all Dockerfiles

Reply:
Make sense. I'll work on that.

Reply:
Fixed